cmd/cloud-controller-manager/controller-manager.go

```diff
@@ -35,7 +35,11 @@ import (
 )
 
 func main() {
-	rand.Seed(time.Now().UnixNano())
+	err := apirand.InitMathRand()
+	if err != nil {
+		klog.Fatalf("cannot seed random: %v", err)
```

Comment:
do we need to call `logs.InitFlags` before logging?

Reply:
@tedyu:

^ @liggitt 's question is still pending:
> do we need to call logs.InitFlags before logging?

this needs to be moved before `apirand.InitMathRand()`
```
	logs.InitLogs()
	defer logs.FlushLogs()
```

you can also try building kubectl locally from this PR using `make all WHAT=cmd/kubectl` and running it.